tests/unit_tests/sensitivity_analysis/test_design_matrix.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from ert.config import ErtConfig
+from ert.sensitivity_analysis.design_matrix import (
+    initialize_parameters,
+    read_design_matrix,
+)
+from ert.storage import open_storage
+
+
+@pytest.mark.usefixtures("copy_poly_case")
+def test_design_matrix(copy_poly_case):
+    config_text = """
+QUEUE_SYSTEM LOCAL
```

Comment:
Suggest just appending `DESIGN_MATRIX design_matrix.xlsx` to the regular `poly.ert`